src/EntityId.js

```diff
@@ -107,16 +65,15 @@ wb.datamodel.EntityId = util.inherit(
 	 * @inheritdoc
 	 */
 	getSortKey: function() {
-		return this._entityType + this._numericId;
+		return this._serialization;
 	},
 
 	/**
 	 * @inheritdoc
 	 */
 	toJSON: function() {
 		return {
-			'entity-type': this._entityType,
-			'numeric-id': this._numericId
```

Comment:
I was considering that, too, but it would be difficult since this data is not present anymore.